python/tk_multi_workfiles/browser_form.py

```diff
@@ -221,9 +221,13 @@ def set_models(self, my_tasks_model, entity_models, file_model):
             self._file_model.set_users(self._file_filters.users)
 
             # add an 'all files' tab:
-            self._add_file_list_form("All", "All Files", show_work_files=True, show_publishes=True )
-            self._add_file_list_form("Working", "Work Files", show_work_files=True, show_publishes=False)
-            self._add_file_list_form("Publishes", "Publishes", show_work_files=False, show_publishes=True)
+            preferred_tabs = app.get_setting("file_browser_tabs")
+            if "All" in preferred_tabs:
```

Comment:
Instead of looking for values that work, it would be great it we could loop over all of them and compare them using `lower` on the user's input to make it a bit more permissive. For entries that do not map to any valid values, we should log a warning using a logger's `warning` method.  You can check out our doc on how to [get a Toolkit logger](https://developer.shotgunsoftware.com/tk-core/utils.html#logging-from-within-your-app-engine-or-framework) and then use it.